src/providers/Publix/Appointments.js

```diff
@@ -152,13 +152,14 @@ class Appointments {
     }
 
     const vaccineTypes = [];
-    if (/appointments for .*(Johnson|Janssen)/i.test(bodyText)) {
+    const vaccineButtons = $body(".vaccineButtons").text();
+    if (/Johnson|Janssen/i.test(bodyText)) {
       vaccineTypes.push("Johnson & Johnson");
     }
-    if (/appointments for .*Moderna/i.test(bodyText)) {
+    if (/Moderna/i.test(bodyText)) {
       vaccineTypes.push("Moderna");
     }
-    if (/appointments for .*Pfizer/i.test(bodyText)) {
+    if (/Pfizer/i.test(bodyText)) {
```

Comment:
This seemed like the smallest reasonable change, but the page appears to determine actual availability dynamically now by loading a URL like:

```
https://www.publix.com/covid-vaccine/${state}/${state}-include.txt?t=${Date.now()}
```

e.g. https://www.publix.com/covid-vaccine/south-carolina/south-carolina-include.txt?t=1622471493108

The format is… interesting, but at least the file has text instructions for how to parse in it. 🤷 

So that’s probably the right change to make after a smaller hotfix like the code here.